modules/framework/galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/ITestMethodResult.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework;
+
+public interface ITestMethodResult {
+    String getMethodName();
+    boolean isPassed();
+    boolean isFailed();
+    Throwable getThrowable();
+}
```

Comment:
Given that this interface can be used directly by tests, we could add some javadoc comments to the interface and the methods?

We could then point people to look at the published javadoc if they want to know more about this.

Reply:
Thanks Eamonn, addressed in https://github.com/galasa-dev/galasa/pull/384/commits/b519c133ed1d61f0d0758fb4af64150ac5d6c3e6